PennMobile/General/SwiftUI Views/CustomPopupView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  CustomPopupView.swift
+//  PennMobile
+//
+//  Created by Christina Qiu on 1/26/24.
+//  Copyright © 2024 PennLabs. All rights reserved.
+//
+
+#if canImport(SwiftUI)
+import SwiftUI
+import PennMobileShared
+#endif
```

Comment:
```suggestion
import SwiftUI
import PennMobileShared
```

The `canImport` check shouldn't be necessary, but we can deal with that in a fast follow